packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div aria-labelledby="AddToCart" class="sf-add-to-cart">
+  <div id="AddToCart" aria-label="Add to cart" class="sf-add-to-cart">
```

Comment:
@justyna-13  why you use here aria-label? button has a text content inside, do we need this?

Reply:
as we discussed we need to consider this case :)